packages/core/tsconfig.json

```diff
@@ -18,5 +18,5 @@
         "baseUrl": "./",
         "experimentalDecorators": true
     },
-    "include": ["src"]
+    "include": ["src", "../hooks/src/Context"]
```

Comment:
How about moving this to the shared folder? not sure if we need context or anything as such under Hooks package, correct me if I'm wrong.

Reply:
let me try it. it would make more sense if we have it there.

Reply:
Should't be under stores folder?

Reply:
moved it to stores. thanks guys